.github/workflows/update_release_draft.yml

```diff
@@ -15,7 +15,9 @@ jobs:
   update:
     runs-on: ubuntu-latest
     needs: extract_pipeline_context
-    if: github.event.pull_request.merged == true
+    if: |
+      (github.event.pull_request.merged == true) &&
+        !contains(github.event.pull_request.labels.*.name, 'changelog:skip')
```

Comment:
Will skipping this step at this point prevent the rest of the action from being executed?

Reply:
It skips the `update` job, but not the previous `extract_pipeline_context` one. I'm tackling that separately, as I'd like to convert that job into a composite action to avoid checking out the code multiple times.